src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php

```diff
@@ -150,12 +153,37 @@ public function configureViews(ViewCollection $viewCollection): void
                     ->setOption('tabTitle', 'sulu_media.formats')
                     ->setParent(static::EDIT_FORM_VIEW)
             );
+
             $viewCollection->add(
                 $this->viewBuilderFactory
                     ->createViewBuilder(static::EDIT_FORM_HISTORY_VIEW, '/history', 'sulu_media.history')
                     ->setOption('tabTitle', 'sulu_media.history')
                     ->setParent(static::EDIT_FORM_VIEW)
             );
+
+            $viewCollection->add(
+                $this->viewBuilderFactory
+                    ->createListViewBuilder(static::EDIT_FORM_ACTIVITY_VIEW, '/activity')
+                    ->setTabTitle('sulu_admin.activity')
+                    ->setResourceKey(ActivityInterface::RESOURCE_KEY)
+                    ->setListKey('activities')
+                    ->addListAdapters(['table'])
+                    ->addAdapterOptions([
+                        'table' => [
+                            'skin' => 'flat',
+                            'show_header' => false,
+                        ],
+                    ])
+                    ->disableTabGap()
+                    ->disableSearching()
+                    ->disableSelection()
+                    ->disableColumnOptions()
+                    ->disableFiltering()
+                    ->addResourceStorePropertiesToListRequest(['id' => 'resourceId'])
+                    ->addRequestParameters(['resourceKey' => MediaInterface::RESOURCE_KEY])
+                    ->addRouterAttributesToListRequest(['locale'])
```

Comment:
not sure if this is needed, i think the locale should be sent to the server automatically. think it could event break things to set it like this when changing the locale 🙈